beacon-chain/rpc/validator/status_test.go

```diff
@@ -435,8 +368,8 @@ func TestValidatorStatus_ExitedSlashed(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not get validator status %v", err)
 	}
-	if resp.Status != ethpb.ValidatorStatus_EXITED_SLASHED {
-		t.Errorf("Wanted %v, got %v", ethpb.ValidatorStatus_EXITED_SLASHED, resp.Status)
+	if resp.Status != ethpb.ValidatorStatus_EXITED {
```

Comment:
Same here, would you mind changing the test name?

Reply:
This one is a bit odd in that we now have two old situations (exited and exited_slashed) that both resolve to exited.  I've tweaked the name but obviously can't have the same name for both.  I do want to keep the test in, mainly to ensure that even though the validator is marked as slashed it reaches the appropriate `Exited` state.

Reply:
Got it!